.travis.yml

```diff
@@ -6,14 +6,18 @@ addons:
   apt:
     packages:
     - sshpass
+    - oracle-java8-set-default
 
 env:
   global:
     - secure: "VayIJwnO14YDmIureSnEE8cNw2gEYJ1b4V8F3YDw+aRVgFB8CGf247JmnOYfEEfadxvjkItAmjAmRzwU+u9U9hosSKuBPSelZ9tmgo3g8LpJnaB578ODBRt/CbLOSSyQyAICxIYJjs/pgV/DJXwdWqW4becWNCw0wgGHqiwDNIQ="
     - secure: "Ga/n9txTxjBDFxsyhbthRZyqVTZA1cOmhH5WaQEWoMcG9yU+RvAdEjqdFmUFHE7SArHIKovyzoupQ9FELESeu8/lkbDtin3MZn3Aj6QXHioBSMihGbduHsGK8yJ1WyjDYXC3fN8XdwNQRZrVFq0ocauPkml/w8uauEsiMo3PX8A="
 before_install:
   - cd reference-implementation
+install:
+  - curl -O https://sideshowbarker.net/nightlies/jar/vnu.jar
```

Comment:
This .jar wasn't found because before_install, above, changed the current directory to reference-implementation/, so the jar is there instead. So this should probably be done in `before_install` instead of moving it to `script`?